cypress/e2e/signIn.cy.js

```diff
@@ -19,7 +19,7 @@ describe('Sign In page', () => {
   
   it('should provide an ability to log in with existing credentials', () => {
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
+    cy.registerNewUser(user.email, user.username, user.password);
```

Comment:
The `cy.registerNewUser` command is used here, but it seems like this command is not defined in the provided `commands.js` file. Ensure that this custom command is defined correctly, or replace it with the appropriate logic to register a new user.